packages/block-editor/src/components/block-list/style.scss

```diff
@@ -189,11 +189,6 @@
 			}
 		}
 	}
-
-	// Toolbar is captured
-	&.has-toolbar-captured::before {
-		left: 0; // place "selected" indicator closer to block due to no toolbar
```

Comment:
This rule is targeting `.block-editor-block-list__layout`, but `.has-toolbar-captured` is only added to blocks, so it's broken. I don't see any issues with the block border without this rule. Everything is looking good. Additionally there are plans to get rid of the border altogether anyway.